doc/man/8/cephfs-top.rst

```diff
@@ -71,6 +71,30 @@ Descriptions of fields
 
    number of opened inodes
 
+.. describe:: rio
+
+   total size of read IOs
+
+.. describe:: wio
```

Comment:
`wtot` (write total) and `rtot` (read total) would be more intuitive I think. @vshankar @joscollin what do you think?

Reply:
Looks okay, trivial to me. What about `rtio` and `wtio`?

Reply:
Yeah, the `rtio` and `wtio` seems better for me.